code/_onclick/cyborg.dm

```diff
@@ -15,6 +15,8 @@
 		return
 	changeNext_click(1)
 
+	if(!cell || cell.charge <= 0 || lockcharge) // Disable UIs if the Borg is unpowered or locked.
+		return
```

Comment:
```suggestion
	if(!cell || cell.charge <= 0 || lockcharge) // Disable UIs if the Borg is unpowered or locked.
		return
```

This will prevent attacks as a whole, not just UI interactions so I'm a bit concerned with that change

Reply:
Echoing what mochi said